static/js/settings_config.ts

```diff
@@ -486,6 +487,9 @@ export const display_settings_labels = {
         defaultMessage: "Show user list on left sidebar in narrow windows",
     }),
     starred_message_counts: $t({defaultMessage: "Show counts for starred messages"}),
+    send_private_message_on_invitee_signup: $t({
+        defaultMessage: "Send private message when an invitee join's",
```

Comment:
Should be "joins" instead of "join's". Also I am not sure of the label here, I think it would be better to have `Receive private message when an invitee joins` and the variables in both backend and here accordingly. We should get some more feedback/suggestions here. @alya thoughts?